french version dev/quickstart and developers

AnnaCICI opened the pull request:
dev/quickstart and developers

Relates to #69. Finished:
- ` docs/fr-fr/dev/quickstart.md`
- `docs/fr-fr/developers/developers_dev.md `

AnnaCICI committed:
dev/quickstart and developers

dev/quickstart and developers

chenrui333 committed:
Merge branch 'dev' of github.com:wuhan2020/wuhan2020.github.io into MK-frVersion-developers-quickstart

* 'dev' of github.com:wuhan2020/wuhan2020.github.io:
  Beng french version doc (#140)
  italian translation: update portal contents (#135)
  Fix image path (#137)

chenrui333 reviewed (approved):
看了下 markdown syntax 没啥问题。

chenrui333 committed:
Update quickstart.md

chenrui333 commented:
relates to https://github.com/wuhan2020/wuhan2020.github.io/issues/69

chenrui333 committed:
Merge branch 'dev' into MK-frVersion-developers-quickstart

GindaChen committed:
Fix the rendering of table in developers_dev.md

The rendering of developers_dev.md has a little problem in Github. 
But in Typora it's always good - which I wonder where the problem is.

GindaChen reviewed (approved):
LGTM!
